.github/workflows/compliance.yaml

```diff
@@ -0,0 +1,58 @@
+name: Compliance
+
+on:
+  pull_request_target:
```

Comment:
This is good! It's much safer than `pull_request`.

Reply:
I always need to read and re-read the explanation of the difference between the two...